Openmm compile with error "TestCudaAmoebaAngleForce.cpp(303) : error C2065: 'M_PI' : undeclared identifier" on Windows

Penomenon:
Openmm compile with error "TestCudaAmoebaAngleForce.cpp(303) : error C2065: 'M_PI' : undeclared identifier" on Windows


Environment:
vs2010 sp1
Windows 7 x64
Openmm verson: Branch master latest


Step to reproduce:
1. Install CMAKE 3.4.0 for windows
2. Generate "NMake Makefiles" by CMAKE for vs2010
2. run script "nmake BUILD=Release"


Workaround:
Add  code as below in TestCudaAmoebaAngleForce.cpp
#define _USE_MATH_DEFINES
#include <math.h>

--- a/plugins/amoeba/platforms/cuda/tests/TestCudaAmoebaAngleForce.cpp
+++ b/plugins/amoeba/platforms/cuda/tests/TestCudaAmoebaAngleForce.cpp
@@ -33,6 +33,9 @@
  * This tests the CUDA implementation of CudaAmoebaAngleForce.
  */
 
+#ifdef WIN32
+  #define _USE_MATH_DEFINES // Needed to get M_PI
+#endif
 #include "openmm/internal/AssertionUtilities.h"
 #include "openmm/Context.h"
 #include "openmm/CustomAngleForce.h"

